### ISSUE TITLE
Snappi : Getting error if flow is created with device endpoints

### ISSUE THREAD

**Issue by ANISH-GOTTAPU** [open-traffic-generator/ixnetwork#246]
Flow is getting created with port endpoints but getting error if created with `device` endpoints

Please find the sample test to reproduce the issue

[sample.zip](https://github.com/open-traffic-generator/ixnetwork/files/5903171/sample.zip)

**Comment by alakendu** [comment]
please use different name as Port and Device

```python
tx, rx = (
    config.ports
    .port(name='tx', location=utils.settings.ports[0])
    .port(name='rx', location=utils.settings.ports[1])
)

device1, device2 = config.devices.device(name="tx", container_name=tx.name).\
    device(name="D1rx", container_name=rx.name)
```

**Comment by ashutshkumr** [comment]
fixed as per `snappi[ixnetwork]==0.3.13`
